compactor/src/compact.rs

```diff
@@ -168,9 +172,21 @@ impl Compactor {
             "Number of bytes of Parquet file candidates",
         );
 
-        let compaction_duration: Metric<DurationHistogram> = registry.register_metric(
+        let duration_histogram_options = DurationHistogramOptions::new([
+            Duration::from_millis(100),
+            Duration::from_millis(500),
+            Duration::from_micros(2_000),
+            Duration::from_millis(5_000),
+            Duration::from_millis(15_000),
+            Duration::from_millis(30_000),
+            Duration::from_millis(60_000),  // 1 minute
+            Duration::from_millis(300_000), // 5 minutes
```

Comment:
arithmetic in comments is not checked by the compiler and it often happens that as the code gets change people will forget to change the comments. Prefer this instead:

```suggestion
            Duration::from_millis(5 * 60_000), 
```